src/types/gecko-profile.js

```diff
@@ -0,0 +1,140 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Adding these is making flow freak out on my use of `_toStructOfArrays`.

```
344:   const geckoFrameStruct: GeckoFrameStruct = _toStructOfArrays(
345:     thread.frameTable
346:   );
       ^ object type. Inexact type is incompatible with exact type
344:   const geckoFrameStruct: GeckoFrameStruct = _toStructOfArrays(
                               ^^^^^^^^^^^^^^^^ exact type: object type
```

I think I'd prefer to just move forward with this PR then try to get it to work. I struggled a lot to get it to pass on that operation.